test/controllers/queries_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class QueriesControllerTest < ActionController::TestCase
+  setup do
+
+  end
+
+  test "should execute the graphql query" do
+    post :create, query: '{ __schema }', format: :json
```

Comment:
I guess this could be passed with `head(200)`, but once the suite is running, we can add more tests later